### DISCUSSION TITLE
Check all

### DISCUSSION THREAD

**Discussion by Epitaffio** [Power-Components/livewire-powergrid#750] [Ideas]
# ⚡ PowerGrid ⚡ Feature Request

## Summary

When using the check all checkbox would be nice to ask for checking ALL the rows and not only the showed rows.

Would be quite handy in case we want to do some bulk action after some filters on all the resulset and not only on the current page.

### Suggested Solution(s)

Something like what gmail does:

![immagine](https://user-images.githubusercontent.com/12532609/202730157-0744ecaa-139c-44f6-9509-4e57a17187b3.png)

It show a message with a clickable action link to select all the rows and a count with the number of the selected rows.

Or maybe transform the checkbox in a dropdown to check the current page or everything

**Comment by luanfreitasdev** [comment]
This really is interesting. If you need this feature more urgently, feel free to open a PR :)
